packages/patternfly-4/react-core/src/components/Page/Page.js

```diff
@@ -25,21 +33,80 @@ const defaultProps = {
   children: null,
   className: '',
   header: null,
-  sidebar: null
+  sidebar: null,
+  isManagedSidebar: false,
+  onPageResize: null
 };
 
-const Page = props => {
-  const { className, children, header, sidebar, ...rest } = props;
-  return (
-    <div {...rest} className={css(styles.page, className)}>
-      {header}
-      {sidebar}
-      <main role="main" className={css(styles.pageMain)}>
-        {children}
-      </main>
-    </div>
-  );
-};
+class Page extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Would be a good opportunity to switch this component to class properties & get rid of the constructor:

https://codeburst.io/use-class-properties-to-clean-up-your-classes-and-react-components-93185879f688